component/exporter.go

```diff
@@ -98,6 +98,9 @@ type ExporterCreateParams struct {
 	// Logger that the factory can use during creation and can pass to the created
 	// component to be used later as well.
 	Logger *zap.Logger
+
+	// ApplicationStartInfo can be used by components for informational purposes
+	ApplicationStartInfo ApplicationStartInfo
```

Comment:
This creates a copy per component, alternatively we may declare an interface with read-only functions and pass a pointer to a struct.

Given that this is done only once during initialization, I think it's fine to pass copies, i.e. leave it as it is.